packages/create-waku/src/cli.ts

```diff
@@ -116,7 +117,6 @@ async function init() {
 
   console.log("Setting up project...");
 
-  const templateRoot = path.join(__dirname, "../template");
   const templateDir = path.resolve(templateRoot, chooseProject);
```

Comment:
This can be `path.join`. Not a big deal though.
```suggestion
  const templateDir = path.join(templateRoot, chooseProject);
```